scipy/stats/_continuous_distns.py

```diff
@@ -2923,6 +2923,64 @@ def fit(self, data, *args, **kwds):
 gamma = gamma_gen(a=0.0, name='gamma')
 
 
+class gamma_gompertz_gen(rv_continuous):
+    u""" A Gamma/Gompertz continuous random variable
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `gamma_gompertz` is:
+
+        gamma_gompertz.pdf(x, c, beta) = c * exp(x) * (beta**c) / ((beta - 1 + exp(x))**(c + 1))
+
+    for ``x >= 0``, ``c > 0``, ``beta > 0``.
+
+    `gamma_gompertz` takes ``c`` and ``beta`` as shape parameters.
+
+    %(after_notes)s
+
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Gamma/Gompertz_distribution
+
+    %(example)s
+    """
+
+    def _logsf(self, x, c, beta):
+        em1 = sc.expm1(x)
+        # np.where is used for dealing with the case that x is too large
+        # so that sc.expm1(x) becomes infinity.
+        # Since x is very large in that case, -sc.log1p(em1 / beta)
+        # approximately equals to -sc.log(np.exp(x) / beta) = np.log(beta) - x
+        lw = _lazywhere(np.isfinite(em1), [em1, x, beta],
```

Comment:
see my suggestion below how to rewrite the term

Reply:
Thanks for the comment. Since `sf=(beta/(beta+sc.expm1(x)))**c` or `sf=(1/(1+sc.expm1(x)/beta))**c`, we have `logsf=c(log(beta)-log(beta + sc.expm1(x)))` or `logsf=c*(-log1p(sc.expm1(x)))`.

Reply:
Ok, but then _cdf is incorrect, no? it is implemented as  `1. - (beta ** c) / (beta + sc.expm1(x))` , so the denominator misses **c

Reply:
Thanks for pointing out my errors. My previous CDF and PPF are indeed incorrect. I revise it in the new commit.

Reply:
ok, I think this is fixed